.github/workflows/DemoAppTestFlight.yml

```diff
@@ -0,0 +1,46 @@
+name: Test Flight Deploy DemoApp
+
+on:
+  push:
+    branches: [ main ]
```

Comment:
This should ideally happen only on release not push

Reply:
Right, thank you for the reminder! Changed it to be triggered on release, next time we release we'll see if it works 🙂
It should since I took a trigger from here https://docs.github.com/en/actions/reference/events-that-trigger-workflows#release but who knows